pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -71,6 +71,8 @@ def __call__(self, value, system):
         if 'lang' in self._lowercase_GET_dict:
             self._language = self._lowercase_GET_dict.get('lang')
 
+        self._static_error_message = Config.get('static_error_message').get(self._language)
```

Comment:
If the error message does not exist in the requested language I think this will crash or give an error.
You can avoid this when you use the fallback_lang for this case.

Reply:
Good catch, I thought it was already handled above :+1: